### DISCUSSION TITLE
Self hosted - Create invite?

### DISCUSSION THREAD

**Discussion by damien67** [stoatchat/discussions#133] [Q&A] [answered]
Super exciting project!!

After a bit of struggle I got the stack running without error in my Docker environment. 
I did set the option REVOLT_INVITE_ONLY=1 (trying to create a safe place for my kids and family)...now, how do I create a user/invite??

I went round and round in the documentation but cannot find a way to do it!

**Comment by insertish** [comment, marked as answer]
If you don't mind toying around with the database, currently you create invites by creating new documents in the `invites` collection.

A simple way to do this would be:
```bash
docker-compose exec database mongosh
```

Then insert a new invite code:
```
use revolt
db.invites.insertOne({ _id: "invite code" })
```

Or create several:
```
db.invites.insertMany([{ _id: "a" }, { _id: "b" }])
```

**Reply by DeclanChidlow** [reply]
Fully covered in self-hosting guide as well.

https://github.com/revoltchat/self-hosted/?tab=readme-ov-file#making-your-instance-invite-only
